src/settings.cpp

```diff
@@ -187,9 +187,8 @@ static int ParseIntList(const char *p, int *items, int maxitems)
 			default: {
 				if (n == maxitems) return -1; // we don't accept that many numbers
 				char *end;
-				long v = strtol(p, &end, 0);
+				size_t v = strtoul(p, &end, 0);
 				if (p == end) return -1; // invalid character (not a number)
-				if (sizeof(int) < sizeof(long)) v = ClampToI32(v);
```

Comment:
i'm unsure about this change. it now mimics the behaviour in StringToVal, however i feel like some validation of range should happen.

Reply:
Hmm yes, bit weird to be removing this. Perhaps it should be added to StringToVal?
Condition could probably be sanely removed though..